pages/papers/ruby-human-language.mdx

```diff
@@ -4,38 +4,34 @@ lang: en
+The creator of ruby _Yukihiro "Matz" Matsumoto_ often mentions that he is _"trying to make Ruby natural, not simple"_ complementing this sentence by saying _"Ruby is simple in appearance, but complex inside, like the human body"_ nothing closer to the truth, and do not make mistakes when mentioning human, this being the best way to describe the ruby harmony.
 
-Y es que con solo prestando atencion a los detalles y leyendo con calma las lineas de ruby aunque no sepa programar, entiendo exactamente lo que va a suceder, por ejemplo `5.times { print "Nos *encanta* Ruby -- ¡es fuera de serie!" }` y es que en Ruby todo es un objeto y se demuestra aplicando una accion `times` a un numero `5` es pura magia de ruby.
+And is that just by paying attention to the details and reading the ruby lines calmly but do not know how to program, I understand exactly what is going to happen, for example `5.times {print" We * love * Ruby - it's out serial!" }` and it is that in Ruby everything is an object and it is demonstrated by applying a`times` action to a number `5` it is pure ruby magic.
 
-Hoy compartire con todos ustedes porque ruby es el mejor lenguaje para iniciarse en el mundo de la programacion, y veremos que te sentiras humano escribiendo codigo ruby.
+Today I will share with all of you because ruby is the best language to start in the world of programming, and we will see that you will feel human writing ruby code.
 
 ### Notes
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
  This is where you should explain things such as technical requirements, why you're the best person to speak on this subject, etc...
 </div>
 
-Soy estudiante en Codeable un bootcamp de fullstack web developer donde construimos cosas increibles usando ruby, react y mas. Estoy ilusionado por contar cómo fue mi experiencia en este bootcamp, y como ruby hizo que mi experiencia fuera la mejor de todas en este proceso de aprendizaje y como me ayudo a interiorisar los conceptos fundamentales de programacion. Mi intencion es animar a que cada vez mas personas se inicien en el mundo de la programacion de la mano de en un lenguage tan bello y artistico como lo es ruby.
+I am a student at Codeable, a bootcamp of fullstack web developer where we build amazing things using ruby, react and more. I am excited to tell how my experience was in this bootcamp, and how ruby made my experience the best of all in this learning process and how it helped me to interiorize the fundamental concepts of programming. My intention is to encourage more and more people to start in the world of programming by hand in a language as beautiful and artistic as it is ruby.
```

Comment:
for fullstack web developers* Ruby, React, and more*. I am excited to tell how my experience was in this bootcamp, how  Ruby made my experience the best of all in this learning process, and how it helped me internalize the fundamental concepts of programming. My intention is to encourage more and more people to start in the world of programming hand in hand with a language as beautiful and artistic as it is Ruby.